packages/openapi/src/middleware.test.ts

```diff
@@ -155,8 +155,6 @@ describe('OpenAPI Validator', (): void => {
       ${{ incomingAmount: { value: '-2', assetCode: 'USD', assetScale: 2 } }} | ${'body.incomingAmount.value must match format "uint64"'}                       | ${'invalid incomingAmount, value non-positive'}
       ${{ incomingAmount: { value: '2', assetCode: 4, assetScale: 2 } }}      | ${'body.incomingAmount.assetCode must be string'}                               | ${'invalid incomingAmount, assetCode not string'}
       ${{ incomingAmount: { value: '2', assetCode: 'USD', assetScale: -2 } }} | ${'body.incomingAmount.assetScale must be >= 0'}                                | ${'invalid incomingAmount, assetScale negative'}
-      ${{ description: 123 }}                                                 | ${'body.description must be string'}                                            | ${'invalid description'}
-      ${{ externalRef: 123 }}                                                 | ${'body.externalRef must be string'}                                            | ${'invalid externalRef'}
```

Comment:
I tried to add `metadata` here but this tests uses an old version of the spec (hardcoded on line 39): https://github.com/interledger/open-payments/raw/3930448672cfc678ec2bc02938566a316d83871c/open-api-spec.yaml

I tried to replace with https://github.com/interledger/open-payments/raw/97ad3e99265c3e1eb516eb678178b9f6a4e5065e/openapi/resource-server.yaml but all tests failed 🤷‍♂️